src/pages/PageCallManage/index.tsx

```diff
@@ -212,10 +223,20 @@ class PageCallManage extends React.Component<{
                 isUserCalling={!partyNumber?.includes('+')}
                 callerName={getCallerName}
                 callerNumber={partyNumber}
-                containerStyle={{ marginTop: getCallerName ? '15%' : '40%' }}
+                containerStyle={{
+                  marginTop: getCallerName || showKeyPad ? '15%' : '40%',
+                }}
               />
               {answered && this.renderCallTime()}
-              {this.renderBtns(currentCall)}
+              {!showKeyPad && this.renderBtns(currentCall)}
+              {showKeyPad && (
+                <PageDtmfKeypad
+                  callId={id}
+                  partyName={partyName}
+                  hangup={hangup}
+                  onHidePress={() => this.setState({ showKeyPad: false })}
+                ></PageDtmfKeypad>
+              )}
```

Comment:
Use ternary operator

Reply:
Use ternary operators as I told you on Skype. Use { x ? y : z } instead of { x && y } and { !x && z }.